### PR TITLE
update for osmo-test-5 testnet

### PR DESCRIPTION
**Pull request by TrevorJTClarke**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Thank you. osmo-test-4 still remains the official testnet for Osmosis and thus don't want to remove it. Can we please instead have this added under a new chain folder 'osmosistestnet5'?

**Comment by TrevorJTClarke** [comment]
> Thank you. osmo-test-4 still remains the official testnet for Osmosis and thus don't want to remove it. Can we please instead have this added under a new chain folder 'osmosistestnet5'?

Yeah certainly!! We're utilizing already, but figured it would be good to provide so its ready to merge when 4 is done.

File: testnets/osmosistestnet5/chain.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    ],
+    "cosmos_sdk_version": "0.45",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "cosmwasm_version": "0.30",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "genesis_url": "https://github.com/osmosis-labs/networks/raw/main/osmo-test-4/genesis.tar.bz2"
```
Comment (JeremyParish69) [inline testnets/osmosistestnet5/chain.json:47]: would have a new genesis, no?

File: testnets/osmosistestnet5/chain.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "osmosistestnet",
```
Comment (JeremyParish69) [inline testnets/osmosistestnet5/chain.json:3]: chain_name: "osmosistestnet5"

File: testnets/osmosistestnet5/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "osmosistestnet",
```
Comment (JeremyParish69) [inline testnets/osmosistestnet5/assetlist.json:3]: chain_name: "osmosistestnet5"

File: testnets/osmosistestnet5/chain.json (hunk 1 above)
Comment (TrevorJTClarke) [reply testnets/osmosistestnet5/chain.json]: sorry yeah i missed that, following this doc here: https://github.com/osmosis-labs/testnets/tree/main/testnets/osmo-test-5

updated to the new genesis, seems they are hosting differently now: https://genesis.osmotest5.osmosis.zone/genesis.json

File: testnets/osmosistestnet5/chain.json (hunk 2 above)
Comment (TrevorJTClarke) [reply testnets/osmosistestnet5/chain.json]: fixed

File: testnets/osmosistestnet5/assetlist.json (hunk 3 above)
Comment (TrevorJTClarke) [reply testnets/osmosistestnet5/assetlist.json]: fixed

**Pull request merged**